service/grails-app/services/org/olf/rs/ReshareActionService.groovy

```diff
@@ -94,7 +95,14 @@ public class ReshareActionService {
   public Map notiftyPullSlipPrinted(PatronRequest pr) {
     log.debug("notiftyPullSlipPrinted(${pr})");
     Map result = [status:false];
-    pr
+    if( pr.pickupLocationCode ) {
+      DirectoryEntry entry = ReshareApplicationEventHandlerService
+        .resolveCombinedSymbol(request.JSON.actionParams.pickupLocationCode).owner;
```

Comment:
Would probably use nullsafe navigation here pickupLocaitonCode)?.owner  - otherwise this will Null Pointer Exception on failed lookup